api/tests/routes/backoffice/pro_test.py

```diff
@@ -548,7 +564,7 @@ def test_can_search_venue_by_contact_email(self, authenticated_client):
     def test_can_search_venue_by_name(self, authenticated_client):
         self._create_venues()
 
-        with assert_num_queries(self.expected_num_queries):
+        with assert_num_queries(self.expected_num_queries + 1):  # feature flag
```

Comment:
Ici aussi, mais aussi pour les autres plus bas : 
```suggestion
        with assert_num_queries(self.expected_num_queries_with_ff):
```